src/Components/NavBar/NotificationsBadge.tsx

```diff
@@ -73,6 +74,8 @@ export const NotificationsBadge: React.FC<{
         )
       }}
     />
+  ) : (
+    <CircularCount />
```

Comment:
This is the default view of your badge w/ any cookie'd data, so it's what we should show instead (instead of `null`).